.travis.yml

```diff
@@ -1,5 +1,8 @@
 language: java
 
+jdk:
+  - openjdk8
```

Comment:
running with jdk 11 leads to a compile time error -- the https impl details are no longer accessible there.
Once webtau moves to jdk 11, we need to migrate to the recommended Builder solution.